### ISSUE TITLE
Ad Blocking not working

### ISSUE THREAD

**Issue by BuckleyAlex** [JairusSW/Chess.com#4]
The ad blocking feature isnt working for me. I am using win11 if that means anything to you.

**Comment by JairusSW** [comment]
Yes, I know what is going on, so I'll fix it some time soon. Sorry for not seeing this until now!

**Comment by JairusSW** [comment]
Started working on it again, nearly there. Sorry for the delay, I have been slacking.

**Comment by ChaoticMaximus** [comment]
can the app open offline or is it an emulator of the online version? when i open the app offline it just showsa white screen. id love to be able to play bots or whatever if possible. Also, you're the only guy with this windows app that i know of so kudos to you!
